userguide/image_classifier/README.md

```diff
@@ -30,6 +30,43 @@ data.save('cats-dogs.sframe')
 data.explore()
 ```
 
+#### Annotating Data
+
+If you only have images without corresponding annotations, you can 
+use the annoation utility built into the image_classifier. An example
+of it's usage is shown below with the **PetImages** subfolder above.
```

Comment:
`s/it's/its` -- "it's" is always a contraction for "it is" whereas "its" is the possessive.